src/enkf/CMakeLists.txt

```diff
@@ -73,9 +73,9 @@ if(BUILD_ENKF)
   set(ENKFEXEC "${ENKF_BASE}${debug_suffix}${GSISUFFIX}" CACHE INTERNAL "ENKF Executable name")
   add_executable(${ENKFEXEC} enkf_main.f90)
   target_link_libraries(${ENKFEXEC} enkflib enkfdeplib ${GSILIB} ${GSISHAREDLIB} ${CORE_LIBRARIES}
-  ${MPI_Fortran_LIBRARIES} ${LAPACK_LIBRARIES} ${NETCDF_Fortran_LIBRARIES} ${NETCDF_C_LIBRARIES}
-  ${FV3GFS_NCIO_LIBRARIES} ${HDF5_Fortran_HL_LIBRARIES}
-  ${EXTRA_LINKER_FLAGS} ${HDF5_LIBRARIES} ${CURL_LIBRARIES} ${GSI_LDFLAGS} ${CORE_BUILT} ${CORE_LIBRARIES} ${CORE_BUILT} ${NCDIAG_LIBRARIES})
+  ${MPI_Fortran_LIBRARIES} ${LAPACK_LIBRARIES} ${CORE_LIBRARIES} ${CORE_BUILT}
+  ${NETCDF_Fortran_LIBRARIES} ${NETCDF_C_LIBRARIES} ${FV3GFS_NCIO_LIBRARIES} 
```

Comment:
Does NetCDF_FORTRAN_LIBRARY not find its C counterpart dependency?
If it does, one probably does not need NETCDF_C_LIBRARAIES.

Reply:
WCOSS_D and Orion builds with only NETCDF_Fortran_LIBRARIES in `src/gsi` and `src/enkf` `CMakeLists.txt` failed with numerous `undefined reference` to `nc*` messages.   Therefore, adding NETCDF_C_LIBRARIES back to `CMakeLists.txt`.

Reply:
Yes. C library is required.
I should look at the `FindNetCDF.cmake` file.
The one in the weather model provides the C library automatically when the Fortran is requested.
We can do this in the next iteration.